ddl/partition.go

```diff
@@ -66,8 +70,9 @@ func buildTablePartitionInfo(ctx sessionctx.Context, d *ddl, s *ast.CreateTableS
 		pi.Expr = buf.String()
 	} else if s.Partition.ColumnNames != nil {
 		// TODO: Support multiple columns for 'PARTITION BY RANGE COLUMNS'.
-		if enable && len(s.Partition.ColumnNames) != 1 {
-			return nil, errors.Trace(ErrUnsupportedPartitionByRangeColumns)
+		if len(s.Partition.ColumnNames) != 1 {
+			pi.Enable = false
+			ctx.GetSessionVars().StmtCtx.AppendWarning(ErrUnsupportedPartitionByRangeColumns)
```

Comment:
em...I will got 2 warning as below, 
```sql
tidb > create table t (a int, b int) partition by range columns (a,b) (partition p0 values less than (1,2))
Query OK, 0 rows affected
Time: 0.010s
tidb > show warnings;
+---------+------+---------------------------------------------------+
| Level   | Code | Message                                           |
+---------+------+---------------------------------------------------+
| Warning | 1105 | unsupported partition type, treat as normal table |
| Warning | 1105 | unsupported partition by range columns            |
+---------+------+---------------------------------------------------+
```

Reply:
Yes, if there are more than 1 columns in `partition by range columns`, we currently treat it as normal table and return a warning. @crazycs520